recipes/zephyr-7b-beta/dpo/config_full.yaml

```
@@ -4,34 +4,35 @@ model_name_or_path: alignment-handbook/zephyr-7b-sft-full
 # Data training arguments
 # For definitions, see: src/h4/training/config.py
 dataset_mixer:
-  HuggingFaceH4/ultrafeedback_binarized: 1.0
+  HuggingFaceH4/ultrafeedback_binarized_fixed: 1.0
```

Replace with the original source once we fix the dataset:

```suggestion
  HuggingFaceH4/ultrafeedback_binarized: 1.0
```